starter-code/index.js

```diff
@@ -1,17 +1,66 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.items.length) {
+      throw new Error("OutOfBounds");
+    } else return this.items[pos];
```

Comment:
The else statement is not needed here. You can just write: return this.items[pos]